src/SourceBuild/content/repo-projects/Directory.Build.targets

```
@@ -9,6 +9,9 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <DiskUsageCommand Condition="'$(BuildOS)' == 'windows'">powershell "Get-PSDrive -PSProvider FileSystem"</DiskUsageCommand>
```

```suggestion
    <DiskUsageCommand Condition="'$(BuildOS)' == 'windows'">powershell -NoProfile "Get-PSDrive -PSProvider FileSystem"</DiskUsageCommand>
```

Invoking powershell without `-noprofile` is asking for future chaos ;)